tests/instancetype_test.go

```diff
@@ -41,7 +41,7 @@ import (
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Instancetype and Preferences", decorators.SigCompute, func() {
+var _ = FDescribe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Instancetype and Preferences", decorators.SigCompute, func() {
```

Comment:
Would this actually work? I think ginkgo cli parameters like skip will mess this up

Reply:
Last time I needed something like this I used this (note this is CDI):
https://github.com/kubevirt/containerized-data-importer/pull/2384

Reply:
This came up in https://github.com/kubevirt/kubevirt/pull/10136#issuecomment-1643194689 and I wanted to quickly test it in CI

Reply:
Yep so this will probably get ignored unless you remove some ginkgo params

Reply:
Yup it's passing the following `--label-filter` in that ignore focused containers, cool. 

```
FUNC_TEST_LABEL_FILTER='--label-filter=(sig-compute && !(GPU,VGPU,sig-compute-migrations,exclude-native-ssh))&&(!verify-non-root)&&(!CustomSELinux)'
make functest
```